Cargo.toml

```diff
@@ -12,7 +12,7 @@ categories = ["embedded"]
 license = "0BSD"
 
 [dependencies]
-byteorder = { version = "1.0", default-features = false }
+byteorder = { version = "1.0", default-features = false, features = ["i128"] }
```

Comment:
I couldn't find any docs on a way to enable a dependency feature based on a feature enabled in the parent project when the dependency isn't optional. If there is a way to conditionally enable this feature based on the `ipv6` feature. We should definitely do that.

Reply:
For your reference, you can do `"ipv6" = ["byteorder/i128"]`.